examples/simple/src/main.rs

```diff
@@ -93,12 +95,12 @@ impl ApplicationHandler for SimpleVelloApp<'_> {
         event: WindowEvent,
     ) {
         // Ignore the event (return from the function) if
-        //   - we have no render_state
+        //   - `RenderState` is `Suspended`
```

Comment:
Maybe just "the app is suspended"

Reply:
I believe it is the window which is suspended, not necessarily the whole app?

Reply:
Suspended RenderState is not necessarily suspended app. It is also the default state during initialization.

Reply:
Personally I don't see the point of this comment... it just restates the obvious meaning of the code that directly follows.

Reply:
I've written a more concise comment that addresses the purpose of the code rather than restating it.

Reply:
This `simple` example is designed to be over-commented. It's not my preference, but my preference would be to not have this example at all :)

I agree that the new shorter comment is fine.